### PR TITLE
Update MEO Node ID Error

### PR DESCRIPTION
**Pull request by ntalluri**
https://github.com/Reed-CompBio/spras/issues/189 fixing this issue

### CODE REVIEW

**Comment by ntalluri** [comment]
The way I made generate-inputs test suite would require me to change the format of data1 and data0 to test MEO's update.

File: spras/meo.py [hunk 1]
```diff
@@ -10,6 +10,8 @@
 
 __all__ = ['MEO', 'write_properties']
 
+# replaces all underscores in the node names with unicode seperator
```
Comment (agitter) [inline spras/meo.py:13]:
```suggestion
# replaces all underscores in the node names with unicode seperator
# MEO keeps only the substring up to the first underscore when parsing node names
# https://github.com/agitter/meo/blob/1fe57e8ff3952c494e2b14dfdc563a84596e2fcd/src/alg/Vertex.java#L56-L71
```
Let's document why we need this

**Review by agitter — COMMENTED** [review]
The solution looks good to me.

> The way I made generate-inputs test suite would require me to change the format of data1 and data0 to test MEO's update.

Does this still need to be done?

**Comment by ntalluri** [comment]
> The solution looks good to me.
> 
> > The way I made generate-inputs test suite would require me to change the format of data1 and data0 to test MEO's update.
> 
> Does this still need to be done?

I just added an idea, but the only thing I wasn't able to change is the parse_outputs domino case

**Comment by ntalluri** [comment]
localized the testing to be done only in generate inputs instead of it being coupled to the changes in inputs and the config.yaml file used as the examples.

**Pull request merged**